tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -9,7 +9,7 @@
 
 import host_tools.logging as log_tools
 
-MAX_STARTUP_TIME_CPU_US = {'x86_64': 8000, 'aarch64': 12000}
+MAX_STARTUP_TIME_CPU_US = {'x86_64': 8000, 'aarch64': 2500}
```

Comment:
Is it `2500 us` even on somewhat loaded m6g systems?

Reply:
I did not do any kind of stress testing, I just ran the `tests/performance/process_startup_time.py` 100 times and obtained around 2300 ms. I guess that we can do this type of test (and also enable it to run periodically) on both x86_64 and aarch64 once the framework for doing so is available.